monitor/v2.1.10/init.sh

```diff
@@ -32,6 +32,17 @@ if $TIDB_ENABLE_BINLOG
 then
     cp /tmp/binlog.json $GF_PROVISIONING_PATH/dashboards
     sed -i 's/Test-Cluster-Binlog/'$TIDB_CLUSTER_NAME'-Binlog/g'  $GF_PROVISIONING_PATH/dashboards/binlog.json
+else
+    rm -rf /tmp/binlog*.yml
+fi
+
+# Lighting
+if $TIDB_ENABLE_IMPORTER
```

Comment:
We can load all dashboards to simplify the script, otherwise when users deploy lightning and binlog after the cluster is running, the monitor needs to be restarted to load the dashboard.

Reply:
Done